ruby_programming/files_and_serialization/project_event_manager.md

```diff
@@ -732,7 +732,7 @@ The Civic Information API allows registered individuals (registration is free) t
 > government. During supported elections, you can also look up polling places, early vote location, candidate data,
 > and other election official information.
 
-### Accessing the API
+#### Accessing the API
 
 Take a close look at this sample URL for accessing the Civic Information API:
```

Comment:
Everything looks good except this heading is not displaying correctly in the [Lesson Preview Tool](https://www.theodinproject.com/lessons/preview):

![image](https://user-images.githubusercontent.com/85733202/159107856-7202c0e4-5429-49b1-8df7-a6e8925ed8bc.png)